src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -570,8 +570,8 @@
   <data name="TupleElementLocationCountMismatch" xml:space="preserve">
     <value>If tuple element locations are specified, the number of locations must match the cardinality of the tuple.</value>
   </data>
-  <data name="TuplesNeedAtLeastTwoElements" xml:space="preserve">
-    <value>Tuples must have at least two elements.</value>
+  <data name="TuplesNeedAtLeastOneElement" xml:space="preserve">
+    <value>Tuples must have at least one element.</value>
```

Comment:
> I recommend rewording to describe the error in terms of the argument the ArgumentException is reporting a problem for. E.g. "elementTypes may not be empty".

I think this would work too.